Core/include/Acts/TrackFitting/detail/GsfUtils.hpp

```diff
@@ -124,11 +124,8 @@ class ScopedGsfInfoPrinterAndChecker {
                  << stepper.direction(state.stepping).transpose()
                  << " and momentum " << stepper.momentum(state.stepping)
                  << " and charge " << stepper.charge(state.stepping));
-    ACTS_VERBOSE("Propagation is in "
-                 << (state.stepping.navDir == NavigationDirection::Forward
-                         ? "forward"
-                         : "backward")
-                 << " mode");
+    ACTS_VERBOSE("Propagation is in " << state.stepping.navDir.toString()
```

Comment:
The outstream operator should be defined here right?

Reply:
true. will change that